10up/ElasticPress
# Display sync history

### Description of the Change
Displays the sync history on the sync page. Currently limited to Completed syncs (with errors or not).

![elasticpress test_wp-admin_admin php_page=elasticpress-sync](https://github.com/10up/ElasticPress/assets/4100733/6dddd18d-eb30-4308-9d99-b423041b7a6e)

### How to test the Change
- Visit _ElasticPress > Sync_. A _Sync history_ panel should be visible with at least one previous sync (if coming from < 5.0.0).
- When a new sync is completed it should be added to the list automatically without a refresh.
- If a a sync is triggered via WP CLI that should be reflected in the the sync title.
- The history should not appear for the initial sync on a site, or after deleting an index.
- The _Delete and sync..._ and _Advanced options_ panels should only be visible if there is a sync history.

### Changelog Entry
Added - Added a sync history to the Sync page.


### Credits
Props @JakePT @felipeelia @brandwaffle @anjulahettige Apurv Ray


### Checklist:
